runtime/moonbeam/src/precompiles.rs

```diff
@@ -113,6 +115,7 @@ pub type MoonbeamPrecompiles<R> = PrecompileSetBuilder<
 				PrecompileAt<AddressU64<2054>, XcmTransactorWrapper<R>>,
 				PrecompileAt<AddressU64<2055>, AuthorMappingWrapper<R>>,
 				PrecompileAt<AddressU64<2056>, BatchPrecompile<R>, LimitRecursionTo<2>>,
+				PrecompileAt<AddressU64<2059>, XcmUtilsWrapper<R, XcmExecutorConfig>>,
```

Comment:
I already took proxy at `2059`. Mind moving it to `2060`? :D

Reply:
will do